### PR TITLE
Feature: stylise views

### PR DESCRIPTION
**Pull request by SyakiraAzmi**
**What's inside this PR?**
- Add bootstrap gem
- Stylised views of customer, waiter and order

**How to test?**

**Trello card**

### CODE REVIEW

File: app/controllers/customers_controller.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ def index
   # GET /customers/1
   # GET /customers/1.json
   def show
+        @orders = @customer.orders
```
Comment (zaimramlan) [inline app/controllers/customers_controller.rb:13]: @SyakiraAzmi Why did this line was removed?

Comment (SyakiraAzmi) [reply app/controllers/customers_controller.rb:13]: It's added, not removed.

Comment (zaimramlan) [reply app/controllers/customers_controller.rb:13]: Whoops, my bad. Hehe

**Comment by zaimramlan** [comment]
@SyakiraAzmi Looks good!

The only problem is that there are 2 application files `application.css` and `application.scss`. Consider deleting the `.css` one as it is unused and may cause confusions and styling problems in the future.

Simply make the change, commit and push it back to the same branch 😁

**Pull request merged**